package.json

```diff
@@ -8,7 +8,8 @@
     "start": "II_FETCH_ROOT_KEY=1 vite",
     "host": "II_FETCH_ROOT_KEY=1 vite --host",
     "showcase": "II_FETCH_ROOT_KEY=1 vite --mode showcase",
-    "build": "vite build",
+    "build": "vite build && tsc --noEmit",
```

Comment:
I think we could swap these (i.e. start with `tsc`) since there's no point in building if typescript fails. Worse even, if for some reason the build is canceled before ts is run, the dist will exist but it may not be well typed.

Note: that's also what astro suggest: https://docs.astro.build/en/guides/typescript/#type-checking

Reply:
Sure